limitador/benches/bench.rs

```diff
@@ -2,18 +2,21 @@ use criterion::{black_box, criterion_group, criterion_main, Bencher, BenchmarkId
 use rand::seq::SliceRandom;
 
 use limitador::limit::Limit;
+use limitador::storage::disk::{DiskStorage, OptimizeFor};
+use limitador::storage::in_memory::InMemoryStorage;
 use limitador::storage::CounterStorage;
 use limitador::RateLimiter;
 use rand::SeedableRng;
 use std::collections::HashMap;
 use std::fmt::{Display, Formatter};
+use tempdir::TempDir;
 
 const SEED: u64 = 42;
 
 #[cfg(not(feature = "redis"))]
```

Comment:
Oh you mean add it as a `feature`? 🤔 
Unsure… what do you think the value would be? disk is "self sufficient" in the sense that it doesn't require an additional service... Now, we could do it to keep compile time and size of project actually using it as a dependency (crate) - but those don't exist as of now… 
Anyways, absolute willing to add it as one.

Reply:
No, it's a feature already, because of `wasm` 🤦 
So, yeah, confused… Do you mean skip the `bench_disk` when disk_storage is not enabled? Cause yeah, right now to run `cargo bench`. I had added `redis` to be skipped, because of the external dependency… Which in this case isn't a thing… I'm on a fence, as I'd rather have it tested and any regression being caught, as again, it's self sufficient. But … why not.

Reply:
Maybe I misunderstood the meaning of 

```rust
#[cfg(not(feature = "redis"))]                                   
criterion_group!(benches, bench_in_mem, bench_disk);             
#[cfg(feature = "redis")]                                        
criterion_group!(benches, bench_in_mem, bench_disk, bench_redis);
```

When doing benchmarks, I expect by default to have redis, memory and disk benchmarks. Thus, if limitador was configured to disable `redis_storage`, then benchmarks with redis would not be done. Then, I would expect something similar for the disk storage. If limitador is configured to disable `disk_persistence`, then benchmarks on disk would not run.